src/contact.ts

```diff
@@ -107,20 +110,103 @@ export class Contact implements Sayable {
     }
   }
 
+  /**
+   * Get the weixin number from a contact
+   * Sometimes cannot get weixin number due to weixin security mechanism, not recommend.
+   * @returns {string | ''}
+   *
+   * @example
+   * ```ts
+   * const weixin = contact.weixin()
+   * ```
+   */
   public weixin()   { return this.obj && this.obj.weixin || '' }
+
+  /**
+   * Get the name from a contact
+   *
+   * @returns {string | ''}
+   *
+   * @example
+   * ```ts
+   * const name = contact.name()
+   * ```
+   */
   public name()     { return UtilLib.plainText(this.obj && this.obj.name || '') }
+
+  /**
+   * Check if contact is stranger
+   *
+   * @returns {boolean} True for not friend of the bot, False for friend of the bot
+   *
+   * @example
+   * ```ts
+   * const isStranger = contact.stranger()
+   * ```
+   */
   public stranger() { return this.obj && this.obj.stranger }
+
+  /**
+   * Check if the contact is star contact.
+   *
+   * @returns {boolean} True for star friend, False for no star friend
+   *
+   * @example
+   * ```ts
+   * const isStar = contact.star()
+   * ```
+   */
   public star()     { return this.obj && this.obj.star }
```

Comment:
public star(): boolean

and make sure the function is return a type of boolean